server/datastore/mysql/mysql_test.go

```diff
@@ -90,6 +90,8 @@ func TestSanitizeColumn(t *testing.T) {
 		{"foobar*baz", "`foobarbaz`"},
 		{"....", ""},
 		{"h.id", "`h`.`id`"},
+		{"id;delete from hosts", "`iddeletefromhosts`"},
+		{"select * from foo", "`selectfromfoo`"},
```

Comment:
Apologies for this unrelated change, but since it came in Slack while working on it, I added a couple test cases to ensure that the `order_key` argument in API calls did not introduce a SQL injection risk (the `order_key` is sanitized by a call to `sanitizeColumn`.